lib/cisco_node_utils/bgp.rb

```diff
@@ -60,8 +61,9 @@ def self.routers
 
       hash_final = {}
       asnum = asnum.to_i unless /\d+.\d+/.match(asnum)
-      hash_tmp = { asnum =>
-        { 'default' => RouterBgp.new(asnum, 'default', false) } }
+      hash_tmp = {
+        asnum => { 'default' => RouterBgp.new(asnum, 'default', false) }
+      }
```

Comment:
This is a rubocop 0.34 vs 0.35 change right? Might want to hold off on this since develop is still using 0.34.2